terraform/implementation/ecs/_variable.tf

```diff
@@ -148,3 +148,14 @@ variable "vpc_cidr" {
   type        = string
   default     = "176.24.0.0/16"
 }
+
+variable "ecr_viewer_database_type" {
+  type        = string
+  description = "Type of the ECR repositories"
```

Comment:
Will you update this description and set a description on the other variable? Please be descriptive, this code may be used by our partners.